### ISSUE TITLE
Please add an option to show the directory size in the SIZE column

### ISSUE THREAD

**Issue by Andreas0602** [mucommander/mucommander#464]
Please add an option to show the directory size in the SIZE column (instead of `<DIR>`).

Other file managers on macOS, for example CRAX Commander or ForkLift, have it too, and it's very useful.

**Comment by ahadas** [comment]
@Andreas0602 it is possible already by invoking the `View` (which defaults to `F3`) operation on the directory

**Comment by Andreas0602** [comment]
yes, but it's not automatic (you have to press F3) and it's only for one single directory.

**Comment by ahadas** [comment]
That's true that it needs to be invoke manually on selected directories at the moment.
The problem with doing it automatically is the resource consumption it might require. @Andreas0602 you say other file managers do it automatically, right? is it enabled by default?

**Comment by Andreas0602** [comment]
Yes, CRAX Commander and ForkLift do it automatically. In both apps, there is an option to activate that behavior. And in both apps, after a fresh installation, the default setting of that option is OFF, probably due to the reason you mentioned above (resource consumption). You only need to turn ON this option once, then it will stay ON forever (until you turn it OFF manually).

**Comment by Andreas0602** [comment]
... in both apps, the size calculation seems to be done in a separate thread, so that the app remains responsive all the time while the directory sizes are calculated in the background.

**Comment by ahadas** [comment]
Sure but like in #463 that you've opened, even when such operations are executed in separate thread(s), it can affect the responsiveness in case we trigger too many operations.. :) (I still need to check what actually happens with the search operation though)

**Comment by DragonSerW** [comment]
ahadas hello! Please add option of showing directory sizes by pressing 'cmd + shift + return'!
Cheers!

**Comment by ahadas** [comment]
@DragonSerW hi, you can configure the shortcut in the Preferences -> Shortcuts dialog, just set an alternative shortcut for the 'View' operation

**Comment by jankiefer** [comment]
@ahadas It would be good to have a shortcut which calculates the sizes for all directories in current pane rather than having to select them one-by-one to compare.

**Comment by rodion-m** [comment]
It was very useful when Total Commander by pressing on space, showed the folder size and selected the next folder (so we can hold space and count space of all folders). So, I hope we will see this feature in muComannder too.

Anyways thanks a lot for a great tool!
